list_unsigned_rpms

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+
+from subprocess import check_output, STDOUT, CalledProcessError
+import argparse
+import glob
+
+
+def find_unsigned_packages(target_dir, gpgkey):
+    packages = glob.glob(f"{target_dir}/*.rpm")
+
+    for package in packages:
+        cmd = [
+            'rpm',
+            '--query',
+            '--queryformat',
+            '%{SIGPGP:pgpsig}',
+            package
+        ]
+
+        output = check_output(cmd, universal_newlines=True, stderr=STDOUT)
+
+        if gpgkey.lower() not in output:
+            yield package
+
+
+def handle_args():
+    parser = argparse.ArgumentParser(description='Sign unsigned RPMs in local stage repository')
+    parser.add_argument(
+        'repository_path',
+        help='Path to repository to sign unsigned RPMs'
+    )
+    parser.add_argument(
+        'gpgkey',
+        help='Short form gpgkey for the version being generated'
```

Comment:
I wonder, why not pass the "full" key and let this script adjust the length as it sees fit?

Reply:
I didn't want this script to "interpret" anything and get it wrong. You tell it what the key is you want it to compare against.

Reply:
fair.